build.gradle

```diff
@@ -176,9 +176,10 @@ if (!hasProperty('java9Home') && System.env.containsKey('JAVA_9_HOME')) {
 assert hasProperty('java9Home') : "Set the property 'java9Home' in your your gradle.properties pointing to a Java 9 installation"
 def java9Path = new File(java9Home, 'bin')
 def java9 = [:].withDefault { execName ->
-  def executable = new File(java9Path, execName)
-  assert executable.exists() : "There is no ${execName} executable in ${java9Path}"
-  executable.toString()
+  def unixExec = new File(java9Path, execName)
```

Comment:
Could you raise this in a separate PR, please?

Reply:
Fixed in #311